README.md

```diff
@@ -130,14 +119,14 @@ Synchronous loading is the preferred method to ensure that Optimizely is always
 ```jsx
 import { OptimizelyProvider, createInstance } from '@optimizely/react-sdk';
 
-const optimizely = createInstance({
+const optimizelyClient = createInstance({
   datafile: window.datafile,

```

Comment:
It's interesting we say that initializing from datafile is the 'preferred' method. I guess I would have thought that if synchronous init is important, the preferred method would be to init w/ both datafile & SDK key? docs state:

"When initializing with both the SDK key and datafile, the SDK will use the given datafile to start, then download the latest version of the datafile in the background"